acme/src/acme/errors.py

```diff
@@ -107,6 +107,16 @@ def __init__(self, failed_authzrs: List['messages.AuthorizationResource']) -> No
         self.failed_authzrs = failed_authzrs
         super().__init__()
 
+    def __str__(self) -> str:
+        msg = []
+        for authzr in self.failed_authzrs:
+            msg.append(f'Authorization for {authzr.body.identifier} ' \
```

Comment:
```suggestion
            msg.append(f'Authorization for {authzr.body.identifier.value} ' \
```
`authzr.body.identifier` turns into a string like `Identifier(typ=IdentifierType(dns), value='example.org')`. i think using `.value` to turn this into just the domain (or [in theory the IP address](https://github.com/certbot/certbot/blob/0075104805e2e4fd9cf0e6ec69d80ab203e03655/acme/src/acme/messages.py#L102-L119)) string is nicer, but if you disagree, please feel to free to leave this as is